### DISCUSSION TITLE
Better library for I2C LCD 1602

### DISCUSSION THREAD

**Discussion by MABeatty1978** [micropython/micropython#10429] [Libraries & Drivers]
I'm using an I2C LCD1602 and using the pico_i2c_lcd.py library for it that I got from [here](https://raw.githubusercontent.com/T-622/RPI-PICO-I2C-LCD/main/pico_i2c_lcd.py)

It doesn't seem to support /n.  A mild annoyance as I'm just padding spacing to correct for it, but I was wondering if anyone had a better library they could recommend?

**Comment by robert-hh** [comment]
You could look at this lib: https://github.com/dhylands/python_lcd

**Comment by peterhinch** [comment]
Or go to https://awesome-micropython.com/ and look up HD44780 where you will find several options.

**Comment by rkompass** [comment]
I just corrected a `\n` - bug in my version of dhylands driver. Which is [here](https://github.com/rkompass/LCD_HD44780_PCF8574_mPy). It is a single file and it's memory footprint is only 1.6 K.

**Reply by robert-hh** [reply]
If you think that dhylands driver has to be changed, please make an PR to his repository.

**Reply by rkompass** [reply]
Will check if this bug was already there...      yes... Done.
